.github/workflows/dockerhub_update_readme.yml

```
@@ -0,0 +1,26 @@
+name: "Update DockerHub README"
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "README.md"
+
+jobs:
+    update-readme:
+      runs-on: "ubuntu-latest"
+      steps:
+        - name: "Login to DockerHub"
+          uses: "docker/login-action@v3"
+          with:
+            username: "${{ secrets.DOCKER_HUB_USERNAME }}"
+            password: "${{ secrets.DOCKER_HUB_TOKEN }}"
+
+
```

According to the documentation, there is no need to login to docker hub, but the readme has to be checked out.

```suggestion
    steps:
    - uses: actions/checkout@v4
```